src/test/java/seedu/address/storage/JsonAdaptedAssignmentTest.java

```diff
@@ -16,6 +16,7 @@
     private static final String INVALID_NAME = "R@chel";
```

Comment:
Maybe example of name can be suited to assignment names more but its a minor problem so LGTM! :-)